### DISCUSSION TITLE
Announcement: Introducing PubPub Platform

### DISCUSSION THREAD

**Discussion by gabestein** [knowledgefutures/pubpub#3130] [General]
Hi Folks,

Today we're announcing a new version of PubPub today, [PubPub Platform](https://lnowledgefutures.org/platform), which is designed to let communities build full-stack knowledge infrastructure tailored to their exact needs. It starts by allowing you to create custom Pub types backed by schemas, introduces a Workflow system that can accommodate any process, allows you to run (and, eventually, develop) powerful Actions that allow you to dynamically modify Pubs, and exposes an API so you can publish your end results any way you want. We're super excited about it, mostly because it's based in large part on your feedback over the years. I'm looking forward to sharing much more about it, including the codebase, in the coming weeks.

There are, however, some tradeoffs. As part of the transition, we'll be sunsetting the current version of PubPub, which we're calling PubPub Legacy, at the end of May 2025, and will not be offering a free hosted version of PubPub Platform at this time. This was a difficult decision, but one we made trying to balance the needs for greater customization many of you have expressed over the years with the need to be sustainable so we can offer you solid, trustworthy infrastructure over the long term.

Of course, we're not abandoning anyone. We're offering 5 transition pathways for communities that we'll be developing in the coming months, including the ability to archive content in perpetuity for free. I'm most excited to announce that PubPub Platform will use no proprietary dependencies, from the fonts on up, which will allow us to support self-hosting, a request many of you have shared over the years.

- You can read more about the transition and options we will make available in [our announcement, here](https://www.knowledgefutures.org/updates/pubpub-platform/).
- You can learn more about the new PubPub Platform on our [new website](https://knowledgefutues.org) and in the [technical explainer](https://www.knowledgefutures.org/updates/platform-tech-overview/) written by @JeffPooley and the PubPub Team.

We suspect many of you will have questions and thoughts to share, which is why we're starting this thread. We've also created a [feedback form](https://airtable.com/appv3zsdPJFIfXpdE/pagIAkaSWSmKPxblv/form) for you to share any thoughts about Legacy or Platform privately, and we will be hosting a community call on August 13, which you can [register for here](https://us02web.zoom.us/webinar/register/WN_LPiMjRKDT_KE9QOPcXzxZg#/registration).

Many thanks for your partnership over the years. We look forward to transitioning to this next phase with you.

Gabe
Head of Platform

**Comment by aimeediehl** [comment]
Hi Gabe,

1. When can we see a preview or test environment for v7/PubPub Platform (and more specifically, a sample journal instance [both editorial management and publishing])?
2. Is the $3,500 annual hosting fee option for PubPub Platform per community or per organization?
3. Does "forum-based support" for annual hosting mean that email technical support will no longer be offered?
4. When are KF's 2023 financials going to be made available?

**Reply by gabestein** [reply]
Hey Aimee,

1. We'll have a sandbox community up in the next few weeks. It won't have 100% of what we're planning for full journal support, as we're still working on those pieces, but will give you an idea of what it will support and allow you to start interacting with it.
2. Per organization. On Platform, "Communities" can support multiple sites/workflows/configurations.
3. We'll offer email support, but it will likely be an add on, not included with base levels of hosting. It just hasn't been sustainable. We will, however, be investing in really comprehensive documentation and offering setup support that should make the need for ongoing support lower.
4. Our 2023 audit took a lot longer than expected due to our accounting firm merging with another, but we'll have it up in the next week or so.

**Comment by aurorabaroni** [comment]
Hi, 

I was wondering if it is currently not possible to publish articles either on the old version (PubPub Legacy) or on the new version (PubPub Platform). If so, when will it be possible to publish them?

**Reply by eastofesten** [reply]
Hi Aurora, 

If you have an existing community on PubPub Legacy, you can continue to publish there until our transition timeline is in effect. If not, you can contact our Partnerships team via [our website](https://www.knowledgefutures.org/contact/) about your project.
